plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -610,6 +618,20 @@ private static function maybe_update_db_version() {
 		}
 	}
 
+	/**
+	 * Set the Store ID if not already present.
+	 *
+	 * @since 8.2.0
+	 */
+	public static function maybe_set_store_id() {
+		$store_id = get_option( self::STORE_ID_OPTION, false );
+		if ( ! $store_id ) {
+			$store_id = wp_generate_uuid4();
+			add_option( self::STORE_ID_OPTION, $store_id );
+		}
+		return $store_id;
```

Comment:
This is a public method to enable testing.

Reply:
I'm assuming the only reason why this is returning the store id is because it's being used in unit testing since everywhere in the business logic I see it being retrieved via `get_option`. 

At a glance, I don't think unit tests should require returning the store_id here (or that the method needs to be public). I'd prefer to see this remain private and also act as a setter (no need to return the set value).

Reply:
Yes, that's a good point! I will change it to not return the `store_id` and update the tests accordingly.